pom.xml

```diff
@@ -54,6 +54,18 @@
 				<artifactId>hamcrest</artifactId>
 				<version>${hamcrest.version}</version>
 			</dependency>
+      <dependency>
+			  <groupId>junit</groupId>
+			  <artifactId>junit</artifactId>
+			  <version>4.12</version>
```

Comment:
you should follow the existing code style and put all dependency versions as properties

Reply:
Applied